website/usage/_processing-pipelines/_extensions.jade

```diff
@@ -144,15 +144,15 @@ p
 +code-exec.
     import spacy
 
-    def sbd_component(doc):
+    def sentencizer_component(doc):
```

Comment:
Maybe we should call this `custom_sentencizer` here? Just to make it more clear that this is a custom implementation and not the *actual* sentencizer component you'd add via `nlp.create_pipe`?

```suggestion
    def custom_sentencizer(doc):
```